module/VuFind/src/VuFind/Controller/HoldsTrait.php

```diff
@@ -52,6 +52,13 @@ trait HoldsTrait
     public function holdAction()
     {
         $driver = $this->loadRecord();
+        //Holds on EDS API records require a different ID
+        //this id can be obtained from the getRtacIdentifier method
```

Comment:
Minor style suggestions:

```suggestion
        // Holds on EDS API records require a different ID.
        // This id can be obtained from the getRtacIdentifier method
```